pkg/storaged/dialog.jsx

```diff
@@ -1196,9 +1198,15 @@
             const name = (fsys
                 ? fsys.Devnode
                 : block_name(client.blocks[use.block.CryptoBackingDevice] || use.block));
+            let location = use.location;
+            if (use.usage == "mounted") {
+                location = client.strip_mount_point_prefix(location);
+                if (location === false)
+                    location = _("(Not part of target)");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231213-115423-cbae7212-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)